demo/cmd/bookbuyer/bookbuyer.go

```diff
@@ -76,8 +76,16 @@ func debugServer() {
 		router.HandleFunc(h.path, h.fn).Methods(h.method)
 	}
 	http.HandleFunc("/favicon.ico", func(w http.ResponseWriter, r *http.Request) {})
+
+	// Initialize bookbuyer HTTP server
+	server := &http.Server{
+		Addr:              fmt.Sprintf(":%d", *port),
+		Handler:           router,
+		ReadHeaderTimeout: time.Second * 10,
+	}
```

Comment:
Is this being done just to please the linter warning? I don't see anything wrong with the previous code as it uses the default mux server from the http pkg. If that's the case, can't we simply suppress the lint warning?